functions/api.js

```diff
@@ -0,0 +1,320 @@
=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(e,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(e,"__esModule",{value:!0})},i.t=function(e,a){if(1&a&&(e=i(e)),8&a)return e;if(4&a&&"object"==typeof e&&e&&e.__esModule)return e;var n=Object.create(null);if(i.r(n),Object.defineProperty(n,"default",{enumerable:!0,value:e}),2&a&&"string"!=typeof e)for(var t in e)i.d(n,t,function(a){return e[a]}.bind(null,t));return n},i.n=function(e){var a=e&&e.__esModule?function(){return e.default}:function(){return e};return i.d(a,"a",a),a},i.o=function(e,a){return Object.prototype.hasOwnProperty.call(e,a)},i.p="",i(i.s=66)}([function(e,a,i){"use strict";var n=i(56),t=Object.prototype.toString;function o(e){return"[object Array]"===t.call(e)}function s(e){return void 0===e}function r(e){return null!==e&&"object"==typeof e}function c(e){if("[object Object]"!==t.call(e))return!1;var a=Object.getPrototypeOf(e);return null===a||a===Object.prototype}function p(e){return"[object Function]"===t.call(e)}function l(e,a){if(null!=e)if("object"!=typeof e&&(e=[e]),o(e))for(var i=0,n=e.length;i<n;i++)a.call(null,e[i],i,e);else for(var t in e)Object.prototype.hasOwnProperty.call(e,t)&&a.call(null,e[t],t,e)}e.exports={isArray:o,isArrayBuffer:function(e){return"[object ArrayBuffer]"===t.call(e)},isBuffer:function(e){return null!==e&&!s(e)&&null!==e.constructor&&!s(e.constructor)&&"function"==typeof e.constructor.isBuffer&&e.constructor.isBuffer(e)},isFormData:function(e){return"undefined"!=typeof FormData&&e instanceof FormData},isArrayBufferView:function(e){return"undefined"!=typeof ArrayBuffer&&ArrayBuffer.isView?ArrayBuffer.isView(e):e&&e.buffer&&e.buffer instanceof ArrayBuffer},isString:function(e){return"string"==typeof e},isNumber:function(e){return"number"==typeof e},isObject:r,isPlainObject:c,isUndefined:s,isDate:function(e){return"[object Date]"===t.call(e)},isFile:function(e){return"[object File]"===t.call(e)},isBlob:function(e){return"[object Blob]"===t.call(e)},isFunction:p,isStream:function(e){return r(e)&&p(e.pipe)},isURLSearchParams:function(e){return"undefined"!=typeof URLSearchParams&&e instanceof URLSearchParams},isStandardBrowserEnv:function(){return("undefined"==typeof navigator||"ReactNative"!==navigator.product&&"NativeScript"!==navigator.product&&"NS"!==navigator.product)&&("undefined"!=typeof window&&"undefined"!=typeof document)},forEach:l,merge:function e(){var a={};function i(i,n){c(a[n])&&c(i)?a[n]=e(a[n],i):c(i)?a[n]=e({},i):o(i)?a[n]=i.slice():a[n]=i}for(var n=0,t=arguments.length;n<t;n++)l(arguments[n],i);return a},extend:function(e,a,i){return l(a,(function(a,t){e[t]=i&&"function"==typeof a?n(a,i):a})),e},trim:function(e){return e.replace(/^\s*/,"").replace(/\s*$/,"")},stripBOM:function(e){return 65279===e.charCodeAt(0)&&(e=e.slice(1)),e}}},function(e,a,i){"undefined"!=typeof process&&"renderer"===process.type?e.exports=i(77):e.exports=i(79)},function(e,a){e.exports=require("path")},function(module,exports,__webpack_require__){
```

Comment:
what is this file?

Reply:
This file is for netlify hosting, followed the tutorial